content/events/2021-online-unconference/_index.md

```diff
@@ -0,0 +1,67 @@
++++
+title = "Nordic-RSE online unconference"
+description = "Nordic RSE – Research Software Engineers in the nordics – Unconference 29–30.June #NordiRSEunconf"
++++
+
+# Nordic RSE Unconference 2021
+
+We are planning an online unconference on June 29–30. The program will consist of your contributions and we encourage you to submit a quick abstract for a discussion topic, talk, demonstration or any other type of program you would like to run.
+
+## Dates: June 29-30, 13-16 CET (3 + 3 hours)
+
+## Format and goal and scope
+
+To bring the community of research software engineers together in 2021,
+to exchange ideas and solutions and challenges we plan an informal unconference.
+In the unconference format we encourage participants to shape the program and
+we will provide support and infrastructure for that.
+
+This event is not only limited to the Nordics and not only limited to participants
+who are or identify as RSEs but anybody interested in the interface of research and software and data is welcome.
+
+
+## What contributions we welcome
+
+In principle we welcome any contributions which respect the Code of Conduct and in principle of any length. We will try to arrange a program based on your input.
+
+Here are some examples of possible contributions:
+- Discussion topics
+- Demonstrations (tools you like, software you wrote)
+- Quick workshops or reprohacks (only 3 hours in a day)
+- Talks
+- Workshops and lessons: teach us something new
+- Can also be something unfinished and almost working where you would like to get feedback on
+
+## How to contribute ideas
+
+- You can submit a short abstract as a [GitHub issue](https://github.com/nordic-rse/nordic-rse.github.io/issues/new).
+  <!-- - You can also email us at ... Note that your name and abstract will be published.
+  What is our email?
+  -->
+- But you can also pitch an idea at the event but the earlier we know the easier scheduling for us
```

Comment:
Do we really want to have last minute events or should we plan for Birds of a Feather?

Reply:
What is "Birds of a Feather"? I suspect that is what we mean here, but I have not heard the expression in this context.

I do think it's good if the participants can suggest discussion topics at the event. This happened a lot in the December meetup and worked well. But it requires a bit of time and the days are short, so I would have it at the beginning of each day.

Reply:
This is what is a BoF. 

And I think I misunderstood in your program!  So if I understnad we have from 13:00 - 14:20 pre-planned activities and a call for topics/activities for the later unconference session (14:40). Correct?

Birds of a Feather (BoFs) are informal, self-organized meetups focused on specific topics. They are a great way to meet other like-minded community members and have an in-depth discussion on a topic of interest. Anyone is welcome to propose a BoF. All you need is a title, an organizer, and a brief description.

Reply:
I think your original understanding was closer. I'm starting with the assumption that there are no pre-planned activities.

We would ask for ideas at the beginning of each day and maybe have quick introductions of each topic. Then we start with the actual sessions and each participant can decide which session to join.

People can also submit their ideas as issues before the meeting. We will add them to the agenda and participants get an advance warning about these events.

It is a good idea to add the submissions to the first half of the day. This way there is more time to decide which activities you want to take part in.

Reply:
So your version is better if we get a good number of submissions.